scripts/gatkcondaenv.yml.template

```diff
@@ -19,30 +19,34 @@ channels:
-- conda-forge::python=3.6.10        # do not update
+- conda-forge::python=3.7.8         # do not update
 - pip=20.0.2                        # specifying channel may cause a warning to be emitted by conda
-- conda-forge::mkl=2019.5           # MKL typically provides dramatic performance increases for theano, tensorflow, and other key dependencies
-- conda-forge::mkl-service=2.3.0
-- conda-forge::numpy=1.17.5         # do not update, this will break scipy=1.0.0
+- conda-forge::mkl=2022.1.0         # MKL typically provides dramatic performance increases for theano, tensorflow, and other key dependencies
+- conda-forge::mkl-service=2.4.0
+- conda-forge::numpy=1.21.2         # do not update, this will break scipy=1.0.0
                                     #   verify that numpy is compiled against MKL (e.g., by checking *_mkl_info using numpy.show_config())
                                     #   and that it is used in tensorflow, theano, and other key dependencies
-- conda-forge::theano=1.0.4         # it is unlikely that new versions of theano will be released
+- conda-forge::theano=1.0.5         # it is unlikely that new versions of theano will be released
                                     #   verify that this is using numpy compiled against MKL (e.g., by the presence of -lmkl_rt in theano.config.blas.ldflags)
-- defaults::tensorflow=1.15.0       # update only if absolutely necessary, as this may cause conflicts with other core dependencies
-                                    #   verify that this is using numpy compiled against MKL (e.g., by checking tensorflow.pywrap_tensorflow.IsMklEnabled())
-- conda-forge::scipy=1.0.0          # do not update, this will break a scipy.misc.logsumexp import (deprecated in scipy=1.0.0) in pymc3=3.1
-- conda-forge::pymc3=3.1            # do not update, this will break gcnvkernel
-- conda-forge::h5py=2.10.0          # required by keras 2.2.4
-- conda-forge::keras=2.2.4          # updated from pip-installed 2.2.0, which caused various conflicts/clobbers of conda-installed packages
-                                    #   conda-installed 2.2.4 appears to be the most recent version with a consistent API and without conflicts/clobbers
-                                    #   if you wish to update, note that versions of conda-forge::keras after 2.2.5
-                                    #   undesirably set the environment variable KERAS_BACKEND = theano by default
-- defaults::intel-openmp=2019.4
+
+- conda-forge::pymc3=3.5            # do not update, this will break gcnvkernel
+- conda-forge::scipy=1.5.3
+- conda-forge::h5py=3.1.0
+
+- defaults::intel-openmp=2021.4.0
 - conda-forge::scikit-learn=0.23.1
 - conda-forge::matplotlib=3.2.1
 - conda-forge::pandas=1.0.3
 - conda-forge::typing_extensions=4.1.1   # see https://github.com/broadinstitute/gatk/issues/7800 and linked PRs
 
+- pytorch=1.10.1
+- setuptools=59.5.0
+
+# other python dependencies; these should be removed after functionality is moved into Java code
```

Comment:
If these lines weren't changed, perhaps restore their original location to minimize the diff?